pipelines/nextflow/workflows/additional_seq_prepare/main.nf

```diff
@@ -50,15 +50,20 @@ if (params.help) {
     exit 0
 }
 
-assert params.accession, "Parameter 'acession' is not specified"
+assert params.accession, "Parameter 'accession' is not specified"
 assert params.prefix, "Parameter 'prefix' is not specified"
```

Comment:
Are we allowed to use an empty string? Should we use empty by default?

Reply:
update pipelines/nextflow/modules/genbank/extract_from_gb.nf  to use `""` for prefix.
check  that extract_from_gb  is allowed to have an empty suffix